api/public/cerbos/response/v1/response.proto

```diff
@@ -444,8 +444,29 @@ message InspectPoliciesResponse {
   option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
     json_schema: {description: "Inspect policies response"}
   };
+
+  message Variable {
+    message Source {
+      enum Type {
```

Comment:
I think `Kind` would be a better name because we are talking about variables and "type" could be misinterpreted as the actual data type of the variable. Also, because `type` is a keyword, it'd be a bit painful to work with in code.